arm-sql/2014-04-01/swagger/dataMasking.json

```diff
@@ -0,0 +1,460 @@
+                }
+            }
+        }
+    },
+    "definitions": {
+        "DataMaskingPolicyProperties": {
+            "properties": {
+                "dataMaskingState": {
+                    "type": "string",
+                    "description": "Gets or sets the state of the data masking policy."
+                },
+                "exemptPrincipals": {
+                    "type": "string",
+                    "description": "Gets or sets the list of the exempt principals"
+                }
+            },
+            "required": [
+                "dataMaskingState",
+                "exemptPrincipals"
+            ],
+            "description": "Represents the properties of an Azure SQL Database data masking policy."
+        },
+        "DataMaskingPolicy": {
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/DataMaskingPolicyProperties",
+                    "description": "Represents the properties of the resource."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "#/definitions/SubResource"
+                }
+            ],
+            "description": "Represents an Azure SQL Database data masking policy."
+        },
+        "DataMaskingRuleProperties": {
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "Gets or sets the rule Id."
+                },
+                "ruleState": {
+                    "type": "string",
+                    "description": "Gets or sets the rule state."
+                },
+                "schemaName": {
+                    "type": "string",
+                    "description": "Gets or sets the schema name on which the data masking rule is applied."
+                },
+                "tableName": {
+                    "type": "string",
+                    "description": "Gets or sets the table name on which the data masking rule is applied."
+                },
+                "columnName": {
+                    "type": "string",
+                    "description": "Gets or sets the column name on which the data masking rule is applied."
+                },
+                "maskingFunction": {
+                    "type": "string",
+                    "description": "Gets or sets the masking function that is used for the data masking rule."
+                },
+                "numberFrom": {
+                    "type": "string",
+                    "description": "Gets or sets the numberFrom property of the masking rule."
+                },
+                "numberTo": {
+                    "type": "string",
+                    "description": "Gets or sets the numberTo property of the data masking rule."
+                },
+                "prefixSize": {
+                    "type": "string",
```

Comment:
Please specify the units in description.
should prefixSize and suffixSize be of type integer?